project/sites/ircommission/themes/ircommission_theme/src/scss/components.scss

```diff
@@ -212,10 +212,6 @@ a[href="/news"] {
   border-bottom: 0px;
 }
 
-.search-result{
-  border-bottom: none;
-}
-
 input[type="checkbox"] {
   display: none;
```

Comment:
This will hide all checkboxes on the site? Which checkbox are you trying to hide.

Reply:
Thanks , I have removed this code. This was for a problem with checkboxes showing on the menu links, it now appears to be resolved and the code is unused